backend/app/views/admin/questions/_form.html.erb

```diff
@@ -0,0 +1,45 @@
+<%#
```

Comment:
I was thinking about having this at the application level instead of per entity.
If we do this at the application level, we can reuse the partials and we will have no need to override any other partial anymore.
does it make sense?